### ISSUE TITLE
add cocoapod

### ISSUE THREAD

**Issue by caoer** [itsniper/FTASync#5]
It will be good if FTASync can support cocoapods so it can be installed easily

**Comment by itsniper** [comment]
Good idea. I haven't used cocoapods yet, but I just bookmarked a post about it to see what the buzz is about :). 

I did notice one of the FTASync forks added a podspec though. If you want, give that a try and let me know if it worked. If it works OK I'll pull it in.

**Comment by Ricardo1980** [comment]
It would be nice, specially because this lib has several dependencies.
